mappings/net/minecraft/world/gen/feature/sizeThreeLayersFeatureSize.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_5203 net/minecraft/world/gen/feature/sizeThreeLayersFeatureSize
```

Comment:
capitalization

Reply:
@Boundarybreaker in fact this is not capitalization but you missed a `/` when naming the class.

Reply:
oh yeah I did miss it